stix_shifter_modules/nozomi/README.md

```diff
@@ -0,0 +1,535 @@
+                    "is_reputation_bad": false
+                },
+                "7": {
+                    "type": "x-nozomi-info",
+                    "zone": "ProdNet-A",
+                    "roles": "consumer",
+                    "is_public": false,
+                    "is_node_learned": true,
+                    "is_reputation_bad": false
+                },
+                "8": {
+                    "type": "mac-addr",
+                    "value": "02:02:02:02:02:02"
+                },
+                "9": {
+                    "type": "x-ibm-ttp-tagging",
+                    "extensions": {
+                        "mitre-attack-ext": {
+                            "technique_id": "T1200",
+                            "technique_name": "Hardware Additions",
+                            "tactic_name": "Initial Access"
+                        }
+                    },
+                    "name": "Hardware Additions",
+                    "kill_chain_phases": [
+                        {
+                            "kill_chain_name": "mitre-attack",
+                            "phase_name": "Initial Access"
+                        }
+                    ]
+                }
+            },
+            "first_observed": "2024-01-22T11:59:09.665Z",
+            "last_observed": "2024-01-22T11:59:09.665Z",
+            "number_observed": 1
+        }
+    ],
+    "spec_version": "2.0"
+}
+```
+### Types of Attributes
+
+| Type                      | Description                                | Example                                                                                                            |
+|---------------------------|--------------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| Dictionary fields         | A key value pair attributes.               | {"type_name": "Malformed traffic"}, {""severity": 10}                                                              |
+| List of dictionary fields | A list containing one or more dictionaries | "techniques": [{"id": "T1200", "name": "Hardware Additions"},{"id": "T1071","name": "Application Layer Protocol"}] |
+
+### Observations
+- Operators ==, !=  for some fields are translated to include?, !include?.(For example: x-ibm-ttp-tagging:name, process:parent_ref.binary_ref.name)
+   Example: [x-ibm-ttp-tagging:name != 'Execution'] translates to query=alerts | where properties !include? \"Execution\"
+- The JWT token expires 30 minutes after being created.
+- It is observed that Nozomi API fails if translated query string length is more than MAX_QUERY_LENGTH. To avoid this failure, query is split at 'OR' conditions and multiple requests are made to Nozomi to get the result. However, the part of query joined by 'AND' operator won't be split and 'Query length is too long or Invalid Query' error is returned if the length of that part is beyond MAX_QUERY_LENGTH.
```

Comment:
I tried this and the exception I got wasn't very descriptive. 

 2024-04-05 02:10:06,110 stix_shifter_modules.nozomi.stix_transmission.error_mapper ERROR    failed to map: {'code': None, 'message': 'Expecting value: line 1 column 1 (char 0)'}

Reply:
Sent a mail for this query, Please check.

Reply:
I sent an e-mail in reply to this one.

Reply:
This one is resolved. I just confirmed that the latest commits resolve the issue I was facing.